lua/plugins/filetype.lua

```diff
@@ -102,13 +102,13 @@ vis.ftdetect.filetypes = {
 		ext = { "%.d$", "%.di$" },
 	},
 	dockerfile = {
-		ext = { "Dockerfile" },
+		ext = { "^Dockerfile$" },
```

Comment:
Dockerfiles are allowed to have a prefix, see https://stackoverflow.com/questions/26077543/how-to-name-dockerfiles for examples.

Reply:
Ah, I didn't know that. Updated to handle both cases.